samples/can/README.rst

```diff
@@ -0,0 +1,45 @@
+.. _can-sample:
+
+Controller Area Network
+###########
+
+Overview
+********
+
+This sample demonstrates how to use the CAN API.
+Messages with standard and extended identifiers are sent over the bus triggered
+by a button event.
+Messages are received using message queues and ISRs. Those messages blink LEDs
```

Comment:
To avoid the repetition, how about saying:

    Messages are received using message queues and ISRs, and 
    message reception is indicated by blinking LEDs and output
    on the console.